### PR TITLE
Omnivore model trained on EPIC KITCHENS 100

### PR DESCRIPTION
**Pull request by nikhilaravi**
- Added new torch hub model `omnivore_swinB_epic`
- Added checkpoint file

### CODE REVIEW

File: models/omnivore_model.py [hunk 1]
```diff
@@ -42,26 +42,32 @@ def get_kinetics_head(dim_in: int = 1024, num_classes: int = 400) -> nn.Module:
...
 
 class OmnivoreModel(nn.Module):
-    def __init__(self, trunk: nn.Module, heads: nn.ModuleDict):
+    def __init__(self, trunk: nn.Module, heads: Union[nn.ModuleDict, nn.Module]):
         super().__init__()
         self.trunk = trunk
         self.heads = heads
         self.types = ["image", "video", "rgbd"]
-        assert all([n in heads for n in self.types]), "All heads must be provided"
+        self.base_model = False
```
Comment (mannatsingh) [inline models/omnivore_model.py:50]: nit: let's call it something else, like `multimodal_model`?

File: models/omnivore_model.py [hunk 2]
```diff
@@ -42,26 +42,32 @@ def get_kinetics_head(dim_in: int = 1024, num_classes: int = 400) -> nn.Module:
...
+        if isinstance(heads, nn.ModuleDict):
+            self.base_model = True
+            assert all([n in heads for n in self.types]), "All heads must be provided"
 
-    def forward(self, x: torch.Tensor, input_type: str = "image"):
+    def forward(self, x: torch.Tensor, input_type: Optional[str] = "image"):
```
Comment (mannatsingh) [inline models/omnivore_model.py:55]: The default can be made `None` - otherwise this looks confusing for single head video models. 

And for multimodal models if `input_type` is `None` we use `image` for BC?

**Review by mannatsingh — APPROVED** [review]
Looks good to me, added some minor nits. We should also add this to the README. @rohitgirdhar can you also share the label mapping for epic so we document it somewhere?

**Comment by rohitgirdhar** [comment]
> Looks good to me, added some minor nits. We should also add this to the README. @rohitgirdhar can you also share the label mapping for epic so we document it somewhere?

Here it is: [action_classes.csv](https://github.com/facebookresearch/omnivore/files/8202266/action_classes.csv)
